Directory.Build.props

```diff
@@ -1,3 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
+
+  <PropertyGroup>
+    <Copyright>Copyright &#xA9; 2012 Marcus Bauer &lt;mabako@gmail.com&gt; // Copyright &#xA9; 2019 Git Extensions</Copyright>
```

Comment:
Is there a particular reason to put it here instead in the **only** csproj the repository have?

Reply:
In the target state we'll put all the common info into a directory.build.props
However right now I don't want to deal with this.